gce_workload_cert_refresh/main.go

```diff
@@ -244,6 +247,16 @@ func refreshCreds() error {
 		return nil
 	}
 
+	// Write the config status immediately to the current symlink if it exists. This allows for communicating 
+	// new config errors. E.g., a user can provide wrong config values multiple times. This requires update to 
+	// the config_status without rotating the symlink because it may contain valid credentials.
+	if _, err := os.Stat(symlink); err == nil {
+		logger.Infof("Writing config_status to existing symlink")
+		if err := os.WriteFile(fmt.Sprintf("%s/config_status", symlink), certConfigStatus, 0644); err != nil {
```

Comment:
what is the intent of writing here, then writing again later? the only steps that could fail in between the two writes are creating the new per-run contents dir and then writing the config_status there again. suggest deleting this entire block.